htdocs/lib2/menu.class.php

```diff
@@ -34,7 +36,7 @@ public function __construct()
         // add country-dependent town list for small map
         $country = $login->getUserCountry();
         if ($opt['map']['towns']['enable'] &&
-            isset($build_map_towns_menu) && $build_map_towns_menu &&   // optimization
+            isset($build_map_towns_menu) && $build_map_towns_menu && // optimization
```

Comment:
Inserting two spaces before the comments looks more readeable to me - it betters separates the comment from the code. Looks like some wrong interpretation if PSR-2 rules (one space between operator and the next expression ).

Can you turn off this mofication?